app/assets/javascripts/webauthn.js

```diff
@@ -22,6 +22,18 @@
     }
   };
 
+  var handleHtmlResponse = function(submit, responseError, response) {
```

Comment:
These changes come from: https://github.com/rubygems/rubygems.org/pull/3334 for anyone else looking at it. It made more sense to render html than to return json for password resets.